TODO.rst

```diff
@@ -2,35 +2,20 @@
 TODO
 ====
 
-See also
-'Issue-Tracker <https://github.com/collective/pas.plugins.ldap/issues>`_
+See also `Issue-Tracker <https://github.com/collective/pas.plugins.ldap/issues>`_
+
 
 Milestone 2.0
 -------------
 
-- remove portrait monkey patch
-- add/delete users
-- add/delete groups
-- add flags for readonly groups and users
-- modes for only groups or only users from ldap
-- SSL/TLS configuration TTW
-- creation defaults TTW
-- group in group (depends on: node.ext.ldap: group.groups support)
-- roles from ldap
-- Option on LDAP inspector whether to use query filters from users and groups
-  config
-
-Related TODO
-------------
-
-- Fix ``yafowil.plone`` resource registration for Plone 5
-
-Misc
-----
-
-::
-
-    node.ext.ldap/src/node/ext/ldap/session.py:51:
-    UnicodeWarning: Unicode equal comparison failed to convert both arguments to
-    Unicode - interpreting them as being unequal
-        if queryFilter in ('', u'', None):
+[ ] - remove portrait monkey patch
+[ ] - add/delete users
+[ ] - add/delete groups
+[ ] - add flags for readonly groups and users
+[ ] - modes for only groups or only users from ldap
+[ ] - SSL/TLS configuration TTW
+[ ] - creation defaults TTW
+[ ] - group in group (depends on: node.ext.ldap: group.groups support)
+[ ] - roles from ldap
+[ ] - Option on LDAP inspector whether to use query filters from users and
+      groups config
```

Comment:
This is MarkDown syntax, which PyPI does not support. In this case, for the last line it gives '(ERROR/3) Unexpected indentation.' Also, for MarkDown it should be the other way around: `- [ ]`.

You can use `restview TODO.rst` to check the ReStructuredText parsing of this file. Or `restview --long-description` to check the long description from `setup.py` that would end up on PyPI.
See https://pypi.org/project/restview/

Reply:
The `longdescription` command from `zest.releaser` works too, and is easier to type, but I prefer `restview` these days.

Reply:
I just finish the work on node.ext.ldap 1.0b7, will fix with the next commit defining the node.ext.ldap dependency version